_unittest/test_20_HFSS.py

```diff
@@ -337,12 +335,10 @@ def test_06c_create_linear_step_sweep(self):
                 step_size=step_size,
                 sweep_type="Incorrect",
             )
-        except AttributeError as e:
-            exception_raised = True
             assert (
-                e.args[0] == "Invalid value for `sweep_type`. The value must be 'Discrete', 'Interpolating', or 'Fast'."
+                execinfo.args[0]
+                == "Invalid value for `sweep_type`. The value must be 'Discrete', 'Interpolating', or 'Fast'."
```

Comment:
```suggestion
                == "Invalid value for 'sweep_type'. The value must be 'Discrete', 'Interpolating', or 'Fast'."
```
Here the first was in double backticks so changed to single quote for consistency.